src/containers/BankApi/BankApiBanks/index.tsx

```diff
@@ -18,28 +18,33 @@ const BankApiBanks: FC = () => {
 
       <DocEndpoint endpoint="/banks" method="GET" />
       <RequestResponseSnippet
-        code={`[
-  {
-    "account_number": "5e12967707909e62b2bb2036c209085a784fabbc3deccefee70052b6181c8ed8",
-    "ip_address": "83.168.1.232",
-    "node_identifier": "d5356888dc9303e44ce52b1e06c3165a7759b9df1e6a6dfbd33ee1c3df1ab4d1",
-    "port": 80,
-    "protocol": "http",
-    "version": "v1.0",
-    "default_transaction_fee": "1.0000000000000000",
-    "trust": "100.00"
-  },
-  {
-    "account_number": "db1a9ac3c356ab744ab4ad5256bb86c2f6dfaa7c1aece1f026a08dbd8c7178f2",
-    "ip_address": "74.124.1.68",
-    "node_identifier": "3214108063cda7b259782c57ff8cec343ad2f1ad35baf38c3503db5cf6f3b2f7",
-    "port": 80,
-    "protocol": "http",
-    "version": "v1.0",
-    "default_transaction_fee": "2.5000000000000000",
-    "trust": "98.32"
-  }
-]`}
+        code={`{
```

Comment:
use response of this URL
http://143.110.137.54/banks?limit=2
also add table of the query params

Reply:
I use `/banks?limit=2&offset=4` which has both `limit` and `offset` for example to the Table params.